native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -0,0 +1,252 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.persist.cmd;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.compiler.syntax.tree.SyntaxTree;
+import io.ballerina.persist.PersistToolsConstants;
+import io.ballerina.persist.nodegenerator.BalSyntaxTreeGenerator;
+import io.ballerina.persist.objects.Entity;
+import io.ballerina.persist.objects.GenerateBalException;
+import io.ballerina.projects.DiagnosticResult;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.PackageCompilation;
+import io.ballerina.projects.Project;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.ProjectException;
+import io.ballerina.projects.directory.BuildProject;
+import io.ballerina.projects.directory.ProjectLoader;
+import io.ballerina.tools.text.TextDocument;
+import io.ballerina.tools.text.TextDocuments;
+import org.ballerinalang.formatter.core.Formatter;
+import org.ballerinalang.formatter.core.FormatterException;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+
+/**
+ * Class to implement "persist" command for ballerina.
+ */
+@CommandLine.Command(
+        name = "generate",
+        description = "generate database configurations."
+        )
+
+public class Generate implements BLauncherCmd {
+
+    private static final PrintStream errStream = System.err;
+
+    private String sourcePath = "";
+    public ProjectEnvironmentBuilder projectEnvironmentBuilder;
+
+    private static final String COMMAND_IDENTIFIER = "persist-generate";
+
+    private CommandLine parentCmdParser;
```

Comment:
Remove if unused.

Reply:
Updated in [895edde](https://github.com/ballerina-platform/persist-tools/pull/15/commits/895eddeb28babc700ff8c472fe1745b73ebe9286)